### PR TITLE
1.6.2

### PR DESCRIPTION
**Pull request by RichPrune**
cunt

### CODE REVIEW

**Review by GildedToxin — DISMISSED** [review]

File: overrides/config/incontrol/loot.json [hunk 1]
```diff
@@ -16,11 +16,7 @@
 
 "mob":"minecraft:iron_golem",
 
-"player":true,
-
-"item":"minecraft:iron_ingot",
-
-"itemcount": "0/0/0"
```
Comment (Jonah-Hansen) [inline overrides/config/incontrol/loot.json:23]: no iron at all?? not sure I agree, we'll get people asking why don't golems drop iron but I do get not wanting people making iron farms

**Review by Jonah-Hansen — APPROVED** [review]

File: overrides/config/incontrol/loot.json (hunk 1 above)
Comment (RichPrune) [reply overrides/config/incontrol/loot.json:23]: yea no iron farms, i want to dissuade vanilla op strats at least, sorry for those who want their starting 4 iron

File: overrides/config/ftbquests/quests/chapters/quests.snbt [hunk 2]
```diff
@@ -12399,7 +12388,7 @@
 					type: "item"
 				}
 				{
-					id: "7E4F26D61480AA55"
+					id: "0811D144BE70E7BB"
```
Comment (Jonah-Hansen) [inline overrides/config/ftbquests/quests/chapters/quests.snbt:12391]: whever quest ids change just double check that any other quest that is supposed to reference it or is dependent is still linked properly! should be fine but just double check

**Comment by Jonah-Hansen** [comment]
closes #422 
closes #434
closes #435
closes #436

**Pull request merged**